### DISCUSSION TITLE
Why is there no new version released recently?

### DISCUSSION THREAD

**Discussion by ghost** [red/red#4847] [Q&A] [answered]
I can get v0.6.4 from `red-lang.org/p/download.html`, or, to try out the latest features, install the compiler toolchain with Rebol and run Red from the sources.

I haven't had any significant problems with v0.6.4. But I'm just curious, **why there used to be more than one tagged version released each year, but none since 2019?** Just wondering, is it because of workflow changes or something? Or are there any technical concerns?

**Comment by greggirwin** [comment, marked as answer]
No technical concerns. This past year has thrown a lot of things off and we focused on core work rather than visibility. We've had plans for some time to change the version tagging and release model, which we hope to make happen for 0.6.5.

**Comment by dander** [comment]
I would add that the automated build is usually pretty stable, and a bit simpler than building yourself.
